dsgrid/registry/registry_manager.py

```diff
@@ -494,6 +496,72 @@ def _update_projects_with_datasets(self, updated_datasets, updated_projects):
             if updated and project.config_id not in updated_projects:
                 updated_projects[project.config_id] = project
 
+    @staticmethod
+    def copy(src: Path, dst: Path, make_data_symlinks=False, use_rsync=False, force=False):
+        """Copy a registry to a new path.
+
+        Parameters
+        ----------
+        src : Path
+        dst : Path
+        simple_model : RegistrySimpleModel
+            Filter all configs and data according to this model.
+        make_data_symlinks : bool
+            If True, make symlinks to data files.
+        use_rsync : bool
+            If True, use rsync instead of copy. Useful for testing when the method is called many
+            times. Not available on Windows.
+        force : bool
+            Overwrite dst if it already exists. Only applies if use_rsync is False.
+
+        Raises
+        ------
+        DSGInvalidParameter
+            Raised if src is not a valid registry.
+            Raised if dst exists, use_rsync is False, and force is False.
+
+        """
+        if not {x.name for x in src.iterdir()}.issuperset({"configs", "data"}):
+            raise DSGInvalidParameter(f"{src} is not a valid registry")
+
+        if use_rsync:
```

Comment:
It looks like, the valid option combos for rsync and symlinks are:
- `use_rsync` (True + Whatever for symlinks)
   (In reality it is True + False only since True + True is restricted in code above)
- `no rsync` + `make_data_symlinks` (False + True)
- `no rsync` + `not make_data_symlinks` (False + False)